app/views/invite_requests/show.html.erb

```diff
@@ -1,3 +1,3 @@
 <%= render :partial => 'invite_request', :locals => {:invite_request => @invite_request} %>
 
-<p><%= ts("To check on the status of your invite, go to ")%> <%= link_to ts("the Request an Invite page"), invite_requests_path %> <%= ts(" and enter your email in the space provided!")%></p>
+<p><%= ts("To check on the status of your invitation, go to ")%> <%= link_to ts("the request an gnvitation page"), invite_requests_path %> <%= ts(" and enter your email in the space provided!")%></p>
```

Comment:
Typo -- it says "gnvitation" instead of "invitation." Also, it should probably still be capitalized as "the Request an Invitation page."